pom.xml

```diff
@@ -477,6 +477,20 @@ under the License.
       </properties>
     </profile>
 
+    <profile>
+      <id>Win-amd64</id>
```

Comment:
Just curious, we already have `Win-x86` but `Win-amd64` is different?

Reply:
Yes, they have different cpu architectures. My computer at home is windows with amd 4900 cpu.